### PR TITLE
[nixio] save waveforms of SpikeTrainProxy objects if available.

### PR DESCRIPTION
**Pull request by JuliaSprenger**

### CODE REVIEW

File: neo/io/nixio.py [hunk 1]
```diff
@@ -1167,7 +1167,7 @@ def _write_spiketrain(self, spiketrain, nixblock, nixgroup):
             return
 
         if isinstance(spiketrain, BaseProxy):
-            spiketrain = spiketrain.load()
+            spiketrain = spiketrain.load(load_waveforms=True)
```
Comment (samuelgarcia) [inline neo/io/nixio.py:1170]: It would control this in an option at class init, no ?
Otherwise we can swtich to super slow when writing.

Comment (samuelgarcia) [reply neo/io/nixio.py:1170]: OK

**Comment by samuelgarcia** [comment]
Hi Julia tell me when ready.

**Comment by JuliaSprenger** [comment]
Hi @samuelgarcia Sorry for the delay. Ready from my side :)

**Pull request merged**